src/ray/raylet/local_task_manager.cc

```diff
@@ -370,7 +374,7 @@ bool LocalTaskManager::TrySpillback(const std::shared_ptr<internal::Work> &work,
                                     bool &is_infeasible) {
   auto scheduling_node_id = cluster_resource_scheduler_->GetBestSchedulableNode(
       work->task.GetTaskSpecification(),
-      work->PrioritizeLocalNode(),
+      /*prioritize_local_node*/ true,
```

Comment:
Why was this change needed?

Reply:
Oh, when a node is selected by the cluster task manager and the task is added to that node's local task manager queue. Then we should only spillback if local has no resource (in the other word, we should prefer staying local if possible).